Makefile

```diff
@@ -153,18 +149,16 @@ test-e2e-setup:: build dev-install cluster-create
 
 .PHONY: cluster-create
 cluster-create::
-	[[ "`$(TOOLS_DIR)/kind get clusters`" =~ "$(KIND_CLUSTER)" ]] || $(TOOLS_DIR)/kind create cluster --image="kindest/node:v$(K8S_VERSION)" --name $(KIND_CLUSTER)
-	$(TOOLS_DIR)/kind export kubeconfig --name $(KIND_CLUSTER)
+	[[ "`$(KIND) get clusters`" =~ "$(KIND_CLUSTER)" ]] || $(KIND) create cluster --image="kindest/node:v$(K8S_VERSION)" --name $(KIND_CLUSTER)
+	$(KIND) export kubeconfig --name $(KIND_CLUSTER)
 
 .PHONY: dev-install
 dev-install::
-	$(SCRIPTS_DIR)/fetch kind 0.20.0
 	$(SCRIPTS_DIR)/fetch kubectl $(K8S_VERSION) # Install kubectl AFTER envtest because envtest includes its own kubectl binary
```

Comment:
should kubectl also be bingo'd?

Reply:
it could be, but I couldn't do it as `bingo get k8s.io/kubectl@latest` didn't work for me, so I left it intact.